tests/integration/users.test.js

```diff
@@ -542,6 +542,34 @@ describe('USERS ROUTES - GET /users', () => {
     });
   });
 
+  describe('HOLDING_ADMIN', () => {
+    beforeEach(async () => {
+      authToken = await getTokenByCredentials(holdingAdminFromAuthCompany.local);
+    });
+
+    it('should get all coachs users (company A), role as a string', async () => {
```

Comment:
Pourquoi specifier "role as a string" ? Et company A?
`should get all coachs users from authCompany" suffit non ?

Ici, est-ce que ca ne serait pas pertinent de plutot tester que l'admin holding recupere les coachs d'une structure qui n'est pas la sienne mais qui est rattachée a la meme societe mere  ?

Reply:
done